ui/src/main/webapp/lib/graphql-errors/index.js

```diff
@@ -1,8 +1,8 @@
 const friendlyMessage = {
   UNKNOWN_ENDPOINT: 'No supported sources were found at this location.',
   DUPLICATE_SOURCE_NAME: 'A source with that name already exists. Please choose a different name.',
-  MISSING_REQUIRED_FIELD: 'There are missing required fields.',
-  EMPTY_FIELD: 'There are empty required fields.',
+  MISSING_REQUIRED_FIELD: 'Missing required field.',
+  EMPTY_FIELD: 'Empty required fields.',
   MISSING_KEY_VALUE: `There was a problem with the request to the server: MISSING_KEY_VALUE`,
```

Comment:
They aren't needed, they were accidentally left in from the genericMessage template string below. They can be removed from line 11 for `UNSUPPORTED_ENUM` type as well.